### PR TITLE
Refractor code

### PR DESCRIPTION
**Pull request by Abhishek-Shr**

### CODE REVIEW

File: client/src/components/Backdrop/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+
```
Comment (abhinav941) [inline client/src/components/Backdrop/index.js]: remove extra lines

File: client/src/components/Filter/Filter.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const Filter=(props)=>{
+    return(
+        <div>
+            
+        </div>
+    )
+}
+
+export default Filter;
```
Comment (abhinav941) [inline client/src/components/Filter/Filter.js]: Delete

File: client/src/components/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+import { LoginStyle } from './style';  
+import { Backdrop } from '../Backdrop';
+import { Button } from 'antd';
+import {GooglePlusCircleFilled,TwitterCircleFilled,FacebookFilled,CloseOutlined  } from "@ant-design/icons";
+
+const Login=(props)=>{
+    // console.log(props);
```
Comment (abhinav941) [inline client/src/components/Login/index.js]: remove console

File: client/src/components/ProductStack/ProductItem/index.js [hunk 4]
```diff
@@ -1,48 +1,17 @@
 import React from 'react';
-// import './ProductItem.css';
```
Comment (abhinav941) [inline client/src/components/ProductStack/ProductItem/index.js:2]: add line here

File: client/src/components/ProductStack/style.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+
```
Comment (abhinav941) [inline client/src/components/ProductStack/style.js]: remove extra line

File: client/src/components/Backdrop/index.js (hunk 1 above)
Comment (Abhishek-Shr) [reply client/src/components/Backdrop/index.js]: removed

File: client/src/components/Filter/Filter.js (hunk 2 above)
Comment (Abhishek-Shr) [reply client/src/components/Filter/Filter.js]: deleted

File: client/src/components/Login/index.js (hunk 3 above)
Comment (Abhishek-Shr) [reply client/src/components/Login/index.js]: done

File: client/src/components/ProductStack/ProductItem/index.js (hunk 4 above)
Comment (Abhishek-Shr) [reply client/src/components/ProductStack/ProductItem/index.js:2]: done

File: client/src/components/ProductStack/style.js (hunk 5 above)
Comment (Abhishek-Shr) [reply client/src/components/ProductStack/style.js]: done

**Pull request merged**